Hunting Queries/DnsEvents/DNS_DomainAnomalousLookupIncrease.txt

```diff
@@ -1,39 +1,46 @@
 // Name: DNS Domain anomalous lookup increase
-
+//
 // Description: 
 // Checking for a threefold increase or more of Domain lookup per ClientIP for today based on daily average for the previous week. 
 // This can potentially identify excessive traffic to a given location that could be indicative of data transfer out of your network to a group of systems on the same second level domain.  
 // For example, if one client is sending requests for test1.badguy.com and another client is sending requests for test2.badguy.com, you may not see a high enough count to be interesting. 
 // However, a combination of the requests to badguy.com could have a high enough count to be interesting. 
 // This is only Name lookups, so it would be recommended to review the Firewall\Webproxy logs in relation to the ClientIP making the interesting requests.
-
-// Data Source: DNS Events
-
+// Id: 1d9951b7-51f0-4aa7-af0c-654359aadfff
+//
+// Data Source: #DNSEvents
+//
 // Tags: #C2, #Exfiltration
-
-DnsEvents 
-| where TimeGenerated >= startofday(ago(8d)) and TimeGenerated <= startofday(ago(1d)) //setting to 00:00:00 for the given days ago
+//
+DnsEvents
+ //setting to 00:00:00 for the given days ago
+| where TimeGenerated >= startofday(ago(8d)) and TimeGenerated <= startofday(ago(1d))
 | where SubType == "LookupQuery"
-| extend DayNumberofWeek = tostring(dayofweek(TimeGenerated)) //getting the associated number of the day of the week so we can map to a given day for later parsing if needed
```

Comment:
put comment on own line - cf also next one

Reply:
several other instances of this below

Reply:
Odd, not sure why they were showing as on the same line, it was not in my check in.